README.md

```diff
@@ -12,15 +12,15 @@
 
 ## INTRODUCTION
 
-AzerothCore (AC) is an open source game-server application for World of Warcraft, currently supporting the 3.3.5a game version.
+AzerothCore (AC) is an open-source game-server application for World of Warcraft, currently supporting the 3.3.5a game version.
 
 It is written in C++ and is based on MaNGOS, TrinityCore and SunwellCore.
 
 
 ## WHY AZEROTHCORE?
 
 1. Stability
-1. Authenticity of the content
+1. The authenticity of the content
```

Comment:
no

Reply:
without "The" it's more formal and sounds more like a table of matters, which it is

Reply:
But it is not propper